ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -2667,6 +2667,55 @@ AHU_BYPSSPC_DX3SC_ECON_HT2SC_SFSS_SFVSC:
   - SFSS
   - SFVSC
 
+#AU-SY-ODI types  
```

Comment:
Remove references to specific sites from all files.

Reply:
Change made